libs/sr/db/src/Database/DataEntries/DisplayLightConeEntry.ts

```diff
@@ -0,0 +1,57 @@
+import type { PathKey, RarityKey } from '@genshin-optimizer/sr/consts'
+import { allPathKeys, allRarityKeys } from '@genshin-optimizer/sr/consts'
+import { DataEntry } from '../DataEntry'
+import type { SroDatabase } from '../Database'
+
+export const lightConeSortKeys = ['level', 'rarity', 'name'] as const
+export type LightConeSortKey = (typeof lightConeSortKeys)[number]
+export interface IDisplayLightCone {
+  editLightConeId: string
```

Comment:
This is interesting, I am not sure how much value persisting this state to the database presents. 
The only side effect is that if you had the relic editor open, and refresh the page, it will auto open again. 
Validation for this value looks scuffed as well. I'd advice removing this for now.